packages/contracts-bedrock/src/L2/GasPriceOracle.sol

```diff
@@ -93,7 +159,10 @@ contract GasPriceOracle is ISemver {
                 total += 16;
             }
         }
-        uint256 unsigned = total + overhead();
+        uint256 unsigned = total;
+        if (!isEcotone) {
```

Comment:
Note that with this approach we check the isEcotone flag twice when we call this from `_getL1FeeBedrock/Ecotone`. Thoughts on this tradeoff(alternative being have a separate internal function)